app/assets/stylesheets/common/_navigation.scss

```diff
@@ -134,8 +134,8 @@ footer {
 /* Working indicator */
 
 #working {
-  @include border-radius($radius);
-  @include box-shadow(0 0 5px 0 $light-blue);
+  border-radius: $radius;
```

Comment:
Properties should be ordered background, border, border-radius, box-shadow, color, font-size, font-weight, left, margin-left, padding, position, text-align, top, width, z-index